app/views/organisers_mailer/organisers_add_email.text.slim

```diff
@@ -0,0 +1,5 @@
+| Hello, 
```

Comment:
please change `text.slim` to `slim.html` and use html tags here - look in other mailers for reference :)